adserver/models.py

```diff
@@ -1883,6 +1883,68 @@ def __str__(self):
         return "%s on %s" % (self.advertisement, self.date)
 
 
+class AdvertiserImpression(BaseImpression):
```

Comment:
I'd probably name this a bit more descriptively? `spend` feels pretty jargony, is there a more classic name for this? I can't think of one offhand tho.. `cost` or `expense`, but don't love either. 

```suggestion
class AdvertiserSpendImpression(BaseImpression):
```

Reply:
It isn't really a "spend" or "cost" impression though. While that's one field, the other inherited fields (views, clicks, offers, decisions) are also what this aggregation is about. It really is an aggregation about an advertiser since it is just advertiser details on a daily basis. I don't think this is an improvement.

Reply:
Hrm, I guess that's true. Just feels quite generic, but I guess I'm alright with it for now?